### PR TITLE
Add Lithuanian

### PR DESCRIPTION
**Pull request by lipis**
https://github.com/wireapp/wire-desktop/pull/559

### CODE REVIEW

**Comment by herrmannplatz** [comment]
Pls update moment.js

**Comment by lipis** [comment]
👍

**Comment by herrmannplatz** [comment]
i meant add the language to moment.js

**Comment by lipis** [comment]
I thought I did.. lol :)

**Comment by herrmannplatz** [comment]
in the includes as well /app/page/template/vendor.htm

**Comment by bennycode** [comment]
My friend Goda will be very happy about it. 😁

**Comment by lipis** [comment]
and I'll be happy with some approvals ✅

File: app/script/localization/strings-tr.coffee [hunk 1]
```diff
@@ -37,6 +37,7 @@ z.string.tr.auth_account_create_account = 'Bir hesap oluştur'
 z.string.tr.auth_account_expiration = 'Oturumunuzun süresi dolduğu için çıkış yapıldı. Lütfen tekrar giriş yapın.'
 z.string.tr.auth_account_get_wire = 'Konuşma, arama; fotoğraf, video, müzik ve GIF paylaşmak için basit, gizli ve güvenli mesajlaşma uygulaması.'
 z.string.tr.auth_account_password_forgot = 'Şifremi unuttum'
+z.string.tr.auth_account_public_computer = 'Bu ortak bir bilgisayar'
 z.string.tr.auth_account_sign_in = 'Giriş yap'
 z.string.tr.auth_account_sign_in_email = 'E-posta'
 z.string.tr.auth_account_sign_in_phone = 'Telefon'
```
Comment (bennycode) [inline app/script/localization/strings-tr.coffee:43]: Are you sure that this one is `Telefon` because it's German?!

Comment (lipis) [reply app/script/localization/strings-tr.coffee:43]: This will have to be address on Crowdin.. not here.

Comment (lipis) [reply app/script/localization/strings-tr.coffee:43]: Also 
screen shot 2017-04-06 at 17 56 22

**Pull request merged**